shared/src/commonMain/kotlin/io/customer/shared/tracking/model/TrackingError.kt

```diff
@@ -0,0 +1,16 @@
+package io.customer.shared.tracking.model
+
+import io.customer.shared.tracking.api.model.TrackingErrorReason
+import kotlinx.serialization.SerialName
+
+/**
+ * Response error class to be used when making tracking request. The class will is shared between
+ * network and other layers and is used by background queue to determine next steps for the task.
+ */
+@kotlinx.serialization.Serializable
+internal data class TrackingError(
+    @SerialName("batch_index") val batchIndex: Int?,
+    @SerialName("reason") val reason: TrackingErrorReason,
+    @SerialName("field") val field: String,
+    @SerialName("message") val message: String,
```

Comment:
Don't see need for annotation

Reply:
Responded above, won't be addressing it in this PR.